### ISSUE TITLE
Add support for name ?

### ISSUE THREAD

**Issue by asafyish** [nats-io/nats-top#17]
It will be really useful to have a 'name' column in addition to 'host', something like:

```sh
$ nats-top --name

gnatsd version 0.6.4 (uptime: 31m42s)
Server:
  Load: CPU: 0.8%   Memory: 5.9M  Slow Consumers: 0
  In:   Msgs: 34.2K  Bytes: 3.0M  Msgs/Sec: 37.9  Bytes/Sec: 3389.7
  Out:  Msgs: 68.3K  Bytes: 6.0M  Msgs/Sec: 75.8  Bytes/Sec: 6779.4

Connections: 4
  HOST                 NAME      CID      SUBS    PENDING     MSGS_TO     MSGS_FROM   BYTES_TO    BYTES_FROM  LANG     VERSION SUBSCRIPTIONS
  127.0.0.1:56134      Service1  2        5       0           11.6K       11.6K       1.1M        905.1K      go       1.1.0   foo, hello
  127.0.1.1:56138      Service2  3        1       0           34.2K       0           3.0M        0           go       1.1.0    _INBOX.a96f3f6853616154d23d1b5072
  127.0.0.1:56144      Service1  4        5       0           11.2K       11.1K       873.5K      1.1M        go       1.1.0   foo, hello
  127.0.0.1:56151      Service3  5        8       0           11.4K       11.5K       1014.6K     1.0M        go       1.1.0   foo, hello
```

Obviously, on startup the client will have to report it's name.

**Comment by derekcollison** [comment]
Ther server supports this and most clients can provide a name through the API. I thought we already supported this but @wallyqs will report back.

**Comment by wallyqs** [comment]
Yes, most of the pieces are there to support this, it just needs to be exposed in `/connz` so that we can display it: https://github.com/nats-io/gnatsd/pull/131
Also agree that it would be useful.

**Comment by derekcollison** [comment]
Merged the gnatsd PR, feel free to add in optionally to nats-top.

**Comment by wallyqs** [comment]
Thanks!

**Comment by glycerine** [comment]
+1 for this. It would make nats-top even more useful.

**Comment by wallyqs** [comment]
Fixed via #22
